scheduler/filter/filter.go

```diff
@@ -33,6 +33,7 @@ func init() {
 		&SlotsFilter{},
 		&DependencyFilter{},
 		&AffinityFilter{},
+		&WhitelistFilter{},
 		&ConstraintFilter{},
```

Comment:
I don't see why ordering would matter, since they're all individual filters. Whitelist is not meant to circumvent any other filters of the request, but it's meant to add an additional one, in case a user wants to specify a specific subset of nodes where the rest of the filters would be applied.

I'll go ahead and put it last either way though :)